src/internal/modules/charge-information/lib/helpers.js

```diff
@@ -85,6 +85,23 @@ const findInvoiceAccountAddress = request => {
   return isChargeable ? address : null;
 };
 
+/**
+ * Checks if the new draft charge version has the same start date as an existing charge version
+ * @param {*} request hapi request object
+ * @param {*} draftChargeVersionStartDate the start date of the draft charge version
+ */
+const isOverridingChargeVersion = async (request, draftChargeVersionStartDate) => {
+  try {
+    const { data: chargeVersions } = await services.water.chargeVersions.getChargeVersionsByLicenceId(request.pre.licence.id);
+    return !!chargeVersions.find(version => version.dateRange.startDate === draftChargeVersionStartDate);
+  } catch (error) {
+    if (error.statusCode !== 404) {
+      return error;
+    } else { return false; }
```

Comment:
I'm not sure this is right, we can't guarantee that there are no charge versions with the same start date if we get a 404

Reply:
I will change the water service to return an empty array with http 200 OK which will be a better approach but the scenario remains. An empty array will result in a false because a charge version for the licence with the same start date was not found so no warning will be displayed.